VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -784,6 +786,22 @@ def UriHasSpecialCharacters(self, blobs):
 
         return uriHasSpecialCharacters
 
+    def get_workload_running(self):
```

Comment:
why are we not using PF command ?

Reply:
this is a better approach than using ps commands. also this is being used in guest agent code. so its tried and tested.